First_section.html

```diff
@@ -9,53 +9,293 @@
+          </ul>
+        </div>
+      </section>
+
+      <section class="About_myself">
+        <div class="top">
+          <h1 id="about">About Myself</h1>
+          <p class="text">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h2 class="link">LET'S CONNECT</h2>
+          <div class="icons">
+            <ul class="image">
+              <li>
+                <a href="#">
+                  <img class="social" src="Vector.png" />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img class="social" src="Linkedin icon.png" />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img class="social" src="Vector (1).png" />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img class="social" src="Vector (3).png" />
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <img class="social" src="Vector (2).png" />
+                </a>
+              </li>
+            </ul>
+          </div>
+          <ul class="link_to_resume">
+            <li class="resume"><a href="#">Get my resume</a></li>
+          </ul>
+        </div>
+        <div class="skill_set">
+          <h1 class="language">Languages</h1>
+          <img class="drop" src="union1.png" alt="dropdown" />
+          <ul class="language_list">
+            <ul class="border">
+              <img class="ellipse" src="Ellipse 1.png" alt="" />
+              <li class="code">Javascript</li>
+            </ul>
+            <ul class="border">
+              <img class="ellipse" src="Ellipse 2.png" alt="" />
+              <li class="code">HTML</li>
+            </ul>
+            <ul class="border">
+              <img class="ellipse" src="Ellipse 3.png" alt="" />
+              <li class="code">CSS</li>
+            </ul>
+          </ul>
+          <ul class="frame1">
+            <li class="framework">
+              Frameworks
+              <img
+                class="right_icon"
+                src="Icon - Chevron - Right - Larger.png"
+                alt="right_icon"
+              />
+            </li>
+          </ul>
+          <img class="separator" src="Separator Bottom.png" alt="line" />
+
+          <ul class="frame2">
+            <li class="skills">
+              Skills
+              <img
+                class="icon_chevron"
+                src="Icon - Chevron - Right - Larger.png"
+                alt="right_icon"
+              />
+            </li>
+          </ul>
+          <img class="bottom_separator" src="Separator Bottom.png" alt="line" />
+        </div>
```

Comment:
:red_circle: The technology categories ("Languages", "Frameworks" and "Skills") should be in a ul, and each of them should be in a li tag. As you can see Language is not in a li tag
Kindly wrap the language in a  li tag as follow :arrow_down: 

```
<ul class=""skill_set">
 <li>
     <h1 class="language">Languages</h1>
          <img class="drop" src="union1.png" alt="dropdown" />
          <ul class="language_list">
            <ul class="border">
              <img class="ellipse" src="Ellipse 1.png" alt="" />
              <li class="code">Javascript</li>
            </ul>
            <ul class="border">
              <img class="ellipse" src="Ellipse 2.png" alt="" />
              <li class="code">HTML</li>
            </ul>
            <ul class="border">
              <img class="ellipse" src="Ellipse 3.png" alt="" />
              <li class="code">CSS</li>
            </ul>
          </ul>
   </li>
  <li>paste framework content here</li>
  <li>paste skills content here</li>
</ul>
```

Reply:
Done, thank you for your kind feedback.